### ISSUE TITLE
Check for "JPG" and "PNG"?

### ISSUE THREAD

**Issue by dathbe** [KristjanESPERANTO/MagicMirror-3rd-Party-Modules#27]
The module [here](https://github.com/vincep5/MMM-MyStandings) has an image file, but the image is not showing up.  Possibly because the file extension is capitalized `JPG`?  Perhaps a quick tweak could search for jpg, JPG, png, PNG, jpeg, and JPEG?

**Comment by KristjanESPERANTO** [comment]
It can be bmp, gif, jpg, jpeg, png or webp - upper or lower case don't matter. The problem is that there is no information about a license. We only can use an image if a repository uses a free license.

**Comment by KristjanESPERANTO** [comment]
Checkout the [README](https://github.com/KristjanESPERANTO/MagicMirror-3rd-Party-Modules/blob/main/README.md#how-do-i-get-my-module-presented-perfectly-in-the-new-list) about images:

> - Image
>   - There are two requirements for an image of your module to be displayed:
>     1. There must be a LICENSE file with a free license or in the `package.json` must be a free license in the license field. Take a look at [this package.json](https://github.com/KristjanESPERANTO/MMM-PublicTransportHafas/blob/main/package.json) as an example.
>     2. There must be an image file in your repository.
>   - If there are several images in your repository, one image is selected according to the following logic: First search for an image that contains 'screenshot' or 'example' in it's name and if none is found, take the first image that is found.

**Comment by KristjanESPERANTO** [comment]
Thank you for your tireless work! 💪 :slightly_smiling_face: With over 1000 modules of very different quality, this is a huge task!

Out of curiosity? Do you make all the changes via the browser or do you use a code editor?

I can highly recommend Visual Studio Code, once one has familiarized oneself with it, many things are much easier.

**Comment by dathbe** [comment]
I've been using the web editor for this, though I have VSCodium (VS Code fork).  I just haven't set it up to be able to push things to my github yet.  I'm just learning the command line for git.

And yes, I see how many modules there are.  I figure if several people each take a few, we can clean them up.  I'm mainly interested in who is still maintaining their repos.  There are a lot of modules I'm seeing with the last commit several years ago...which is fine if things work, but you never know.

**Comment by dathbe** [comment]
Which extension(s) for VS Code do you use to integrate with GitHub?

**Comment by KristjanESPERANTO** [comment]
Regarding GitHub integration these two parts are the most import for me: The built-in "Source Control" and the extension "Git Graph".

>  I'm just learning the command line for git.

That's very good to understand how git works. I still use some of the git commands regularly.

Besides the GitHub integration I recommend installing the ESLint and the prettier extension :slightly_smiling_face:

**Comment by dathbe** [comment]
Thanks.  Will look into them.  I created my own new module, and I'd love to get your comments before I post it to the list.  https://github.com/dathbe/MMM-CustomMessage2/.

It's basically the CustomMessage module, but it allows multiple instances of the module to exist at the same time.  

I used a Creative Commons license, so hopefully I did that right.  And I know I need to learn ESLint.  But anything else you see that should be fixed?

Thanks.
